Startup.cs

```diff
@@ -58,20 +75,53 @@ public void ConfigureServices(IServiceCollection services)
             services.AddAsertoAuthorization(options =>
             {
                 Configuration.GetSection("Aserto").Bind(options);
-                options.ResourceMapper = AuthzResourceContext.Instance.ResourceMapper;
+                options.IdentityMapper = AuthzIdentityContext.Instance.IdentityMapper;
             });
             //end Aserto options handling
 
+            var checkResourceRules = new Dictionary<string, Func<string, HttpRequest, Struct>>();
+
+            checkResourceRules.Add("member", (policyRoot, httpRequest) =>
+            {
+                Struct result = new Struct();
+                if (httpRequest.Method == "POST")
+                {
+                    result.Fields.Add("object_id", Value.ForString("resource-creators"));
+                    result.Fields.Add("object_type", Value.ForString("resource-creator"));
+                    result.Fields.Add("relation", Value.ForString("member"));
+                    return result;
+                }
+
+                if (httpRequest.RouteValues.ContainsKey("id"))
```

Comment:
How would the request contain an "id" parameter if the POST route doesn't have any path params?